Frameworks/MQTTnet.NetStandard/Server/MqttClientSessionsManager.cs

```diff
@@ -262,6 +258,7 @@ private MqttConnectReturnCode ValidateConnection(MqttConnectPacket connectPacket
 
                 foreach (var clientSession in _sessions.Values)
                 {
+                    _logger.Verbose($"DispatchApplicationMessageAsync for {clientSession.ClientId} ");
```

Comment:
Please do not use string interpolation or string.format here. The logger will generate the final string if there is a listener for such messages. With string interpolation here there is a overhead I want to avoid.

Reply:
It's a mistake, I added these logs to debug and forgot to delete them;)